### PR TITLE
[PO review] Back Office for Marketplace Operator

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
## PR Description
Add a concept page about Operator Back Office
## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Review by gibmirmilch — COMMENTED** [review]
updated the docs

File: docs/marketplace/user/intro-to-spryker/back-office-for-marketplace-operator.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+* Creates merchants
+* Edits merchants
+* Approves and denies merchants
+* Activates and deactivates merchants
+
+For detailed instructions on managing merchants, see [Managing merchants](/docs/marketplace/user/back-office-user-guides/{{ page.version }}/marketplace/merchants/managing-merchants.html).
+
+## Offer management
+
+In a marketplace, multiple merchants can sell the same product. That's why, instead of creating duplicate products, merchants create offers. Only the offers approved by a marketplace administrator are displayed in the Storefront. In the **Marketplace** > **Offers** section of the Back Office, a marketplace operator manages offers as follows:
```
Comment (mikost7) [inline docs/marketplace/user/intro-to-spryker/back-office-for-marketplace-operator.md:22]: once we say "marketplace administrator", just below "operator". should we simplify it somehow?

**Review by mikost7 — CHANGES_REQUESTED** [review]
see comment

File: docs/marketplace/user/intro-to-spryker/back-office-for-marketplace-operator.md (hunk 1 above)
Comment (gibmirmilch) [reply docs/marketplace/user/intro-to-spryker/back-office-for-marketplace-operator.md:22]: updated to marketplace administrator

**Pull request merged**